test/rpc-test.js

```diff
@@ -0,0 +1,227 @@
+/* eslint-env mocha */
+/* eslint prefer-arrow-callback: "off" */
+
+'use strict';
+
+const assert = require('./util/assert');
+const consensus = require('../lib/protocol/consensus');
+const FullNode = require('../lib/node/fullnode');
+const Network = require('../lib/protocol/network');
+const network = Network.get('regtest');
+
+const node = new FullNode({
+  network: 'regtest',
+  apiKey: 'foo',
+  walletAuth: true,
+  memory: true,
+  workers: true,
+  plugins: [require('../lib/wallet/plugin')]
+});
+
+const {NodeClient, WalletClient} = require('bclient');
+
+const nclient = new NodeClient({
+  port: network.rpcPort,
+  apiKey: 'foo'
+});
+
+const wclient = new WalletClient({
+  port: network.walletPort,
+  apiKey: 'foo'
+});
+
+const {wdb} = node.require('walletdb');
+
+let addressHot = null;
+let addressMiner = null;
+let walletHot = null;
+let walletMiner = null;
+let blocks = null;
+let txid = null;
+let utxo = null;
+
+describe('RPC', function() {
+  this.timeout(15000);
+
+  it('should open node and create wallets', async () => {
+    consensus.COINBASE_MATURITY = 0;
```

Comment:
This would be better in an `before`, and restore it an `after`, that way if test fail, it does not leave the state modified that may break other tests.